<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 242, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="excercise_ques">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 24pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>                                         Intelligent Hospital Assistance System (IHABOT) 
                                                                Health Check</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="temp_b">
        <property name="styleSheet">
         <string notr="true">
QPushButton {
	font: 18pt &quot;MS Shell Dlg 2&quot;;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background-color: rgb(14, 255, 34);
    padding: 15px;
    }

QPushButton:hover {
    background-color: rgb(7, 255, 119);
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 2.3, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
        </property>
        <property name="text">
         <string>Temperature Measure</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pressure_b">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
	font: 18pt &quot;MS Shell Dlg 2&quot;;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background-color: rgb(100, 100, 100);
    padding: 15px;
    }


</string>
        </property>
        <property name="text">
         <string>Pressure and Pulse Measure</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="oxy_b">
        <property name="styleSheet">
         <string notr="true">
QPushButton {
	font: 18pt &quot;MS Shell Dlg 2&quot;;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background-color: rgb(100, 100, 100);
    padding: 15px;
    }


</string>
        </property>
        <property name="text">
         <string>Oxygen Saturation Measure</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="status">
      <property name="styleSheet">
       <string notr="true">font: 75 24pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back_b">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
	font: 18pt &quot;MS Shell Dlg 2&quot;;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background-color: rgb(14, 255, 34);
    padding: 15px;
    }

QPushButton:hover {
    background-color: rgb(7, 255, 119);
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
        </property>
        <property name="text">
         <string>Go Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
